tests/api/test_addon_uploads.py

```diff
@@ -1002,6 +1040,265 @@ def test_extension_invalid_summary(base_url, session_auth):
             ), f'Actual response message was {edit_addon.text}'
 
 
+@pytest.mark.serial
+@pytest.mark.create_session('api_user')
+def test_extension_invalid_homepage(base_url, session_auth):
+    """Try to add some invalid and unaccepted homepage urls for an addon"""
+    addon = payloads.edit_addon_details['slug']
+    invalid_homepage = [
+        '',
+        '.',
+        'abc123',
+        'example.com',
+        'http://not-valid',
+        'www.some-url.org',
+        base_url,
```

Comment:
Do we need to test all of these?

Reply:
As before, yes, we'd like to cover as many edge cases as possible. We need to make sure that the API responds accordingly when provided with invalid or unaccepted values.